.circleci/config.yml

```diff
@@ -628,5 +666,14 @@ workflows:
                 # @todo remove when done
                 - test_deleteme1
 
+  deploy-api-docs-workflow:
+    jobs:
+      - deploy-docs-job:
+          filters:
+            branches:
+              only:
+                - develop
+                - DDP-4548_OpenAPIToGAE
+
```

Comment:
Note that we are only targeting develop for publishing docs. We can setup a custom domain name for server if needed.